### ISSUE TITLE
Populate `bootcamp_run_id` for older runs

### ISSUE THREAD

**Issue by rachellougee** [mitodl/bootcamp-ecommerce#1368]
We added `bootcamp_run_id` as the identifier to the Bootcamp course model in https://github.com/mitodl/bootcamp-ecommerce/issues/1173. At the time it wasn't a required field for older runs. 

It would be nice if we can populate it for older ones so that it's easier to identify runs when we bring data together across different platforms (like MITxOnline and xpro) 

There are around 10 runs that don't have `bootcamp_run_id` on production. Peter said he need to track down some info to do it
![image](https://user-images.githubusercontent.com/3138890/198726468-638af4ea-9881-43b4-b036-bea97e4d0273.png)

**Comment by rachellougee** [comment]
@pdpinch Yes, this is still missing. It's the same 10 courses that have blank bootcamp_run_id. I can't access to Django admin on production. But the link should be something like this:

admin/klasses/bootcamprun/2/change/
admin/klasses/bootcamprun/4/change/
admin/klasses/bootcamprun/10/change/

**Comment by arslanashraf7** [comment]
@pdpinch Would you like an engineer to take a look at this or this is something to be done manually? But we'll need someone to provide the IDs to be added.
